tests/test_msgtopdf.py

```diff
@@ -22,6 +22,19 @@ def test_init_save_path(self):
         email = Msgtopdf("C:/test/email.msg")
         assert email.save_path == pathlib.PurePath("C:/test/email")
 
+        p = pathlib.Path("C:/test/email/email")
+        p.mkdir(exist_ok=True, parents=True)
+        email = Msgtopdf("C:/test/email.msg")
+        assert email.save_path == pathlib.PurePath("C:/test/email (0)")
+
+        p1 = pathlib.Path("C:/test/email (0)")
```

Comment:
Rather than use `"C:/test/email.msg"` here I would look at tempfile in the standard library to use a safe path for creating a test directory, or create a path using the current working directory + "tests/email", rather than the root directory.

Reply:
https://docs.python.org/3/library/tempfile.html